docs/source/admin/quick_howto/index.rst

```diff
@@ -20,19 +20,23 @@ Traffic Control is a complicated system, and documenting it is not trivial. Some
 
 Traffic Ops
 ===========
-
 .. toctree::
 
-	multi_site.rst
-	federations.rst
-	regionalgeo.rst
-	anonymous_blocking.rst
+	multi_site
+	federations
+	regionalgeo
+	anonymous_blocking
 
 Traffic Portal
 ==============
+.. toctree::
+
+	dnssec
+	ds_requests
+	steering
 
+Other
```

Comment:
Why not just a top-level "CDN in a Box" section alongside TO/TP? Burying it in an "Other" section makes it harder to find.

Reply:
I'd say that the sections that are in there today don't really make sense either, especially since we've deprecated the Perl UI. They're just guides on how to use new stuff/features, lumped under the category of whichever UI was used at the time of implementation I'm guessing. I think CIAB is worthy of being at least alongside those. "Other" will just become a dumping ground of random stuff that should really be categorized into something better.

Reply:
I'll make it top-level. Once I finish with my API format overhaul, I'll start working on the how-to guides, and eventually nothing will be under "Traffic Ops", so every distinct guide can be its own top-level entry